setup.py

```diff
@@ -27,7 +26,7 @@
     },
     entry_points={
         'console_scripts': [
-            'ocrd-froc=ocrd_froc.cli.ocrd_cli:cli',
+            'ocrd-froc-recognize=ocrd_froc.cli.ocrd_cli:cli',
```

Comment:
If FROC can deliver both style and text (depending on params), then IMHO the CLI name should be neutral, otherwise a glance at the workflow might raise unwarranted suspicions.

Reply:
I just wanted to avoid deviating from the usual `ocrd-{project}-{functionality}` scheme. It was originally just `ocrd-froc` but if you think that is better, I can change it.